src/containers/Sidebar/SidebarItem.js

```diff
@@ -112,7 +112,7 @@ export const SidebarCollapsibleItem = ({
 								{t(`Sidebar|${item.name}`)}
 							</span>
 
-							<div className="ms-auto"> {/* Added ms-auto to push SidebarBeacon to the end */}
+							<div className=""> {/* Added ms-auto to push SidebarBeacon to the end */}
```

Comment:
Removal of this class may cause issues with `Beacons` (those are the alerting pills). Please validate if that is true. The Beacon should be visible as well and should be at the very end of the row